README.md

```diff
@@ -0,0 +1,66 @@
+![Hyperledger Sawtooth](https://raw.githubusercontent.com/hyperledger/sawtooth-core/master/images/sawtooth_logo_light_blue-small.png)
+
+# Hyperledger Sawtooth Java SDK
+
+## Install Sawtooth SDK from Central Repository
```

Comment:
Good catch, there's 2 SDKs, the Java one and the [Main one](https://sawtooth.hyperledger.org/docs/core/releases/1.0/app_developers_guide/python_sdk_install.html).

This can get confusing for newbies.